plugins/analysis/client/plugin.go

```diff
@@ -63,6 +69,20 @@ func run(_ *node.Plugin) {
 		valuetransfers.Voter().Events().RoundExecuted.Attach(onRoundExecutedClosure)
 		defer valuetransfers.Voter().Events().RoundExecuted.Detach(onRoundExecutedClosure)
 
+		mana.Events().Pledged.Attach(events.NewClosure(func(ev *mana.PledgedEvent) {
+			if len(pledgeEvents) >= config.Node().GetInt(CfgMaxManaEventBufferSize) {
```

Comment:
let's have a `maxManaEventBufferSize` variable instead, which is initialized at the beginning like `conn`. No need to invoke the config plugin every time we receive an event, because this will not change.